ssvc-calc/CISA-Coordinator-v2.0.3.json

```diff
@@ -64,17 +64,17 @@
 		{
 		    "label": "Minimal",
 		    "key": "M",
-		    "description": "Type of harm is \"All\" (Physical, Environmental,Financial,Psychological). The effect is below the threshold for all aspects described in material."
+		    "description": "Type of harm is \"All\" (Physical, Environmental,Financial,Psychological) and the associated Safety Impact Value is \"None\". The effect is below the threshold for all aspects described in material."
 		},
 		{
 		    "label": "Material",
 		    "key": "A",
-		    "description": "Any one or more of the conditions (Physical, Environmental,Financial,Psychological) hold. \"Physical harm\" means \"Physical distress or injuries for users of the system OR introduces occupational safety hazards OR reduction and/or failure of cyber-physical system’s safety margins.\" \"Environment\" means \"Major externalities (property damage, environmental damage, etc.) imposed on other parties.\" \"Financial\" means \"Financial losses that likely lead to bankruptcy of multiple persons.\" \"Psychological\" means \"Widespread emotional or psychological harm, sufficient to be cause for counselling or therapy, to populations of people.\""
+		    "description": "Any one or more of the conditions (Physical, Environmental,Financial,Psychological) hold and the associated Safety Impact Value is \"Major\". \"Physical harm\" means \"Physical distress or injuries for users of the system OR introduces occupational safety hazards OR reduction and/or failure of cyber-physical system’s safety margins.\" \"Environment\" means \"Major externalities (property damage, environmental damage, etc.) imposed on other parties.\" \"Financial\" means \"Financial losses that likely lead to bankruptcy of multiple persons.\" \"Psychological\" means \"Widespread emotional or psychological harm, sufficient to be cause for counselling or therapy, to populations of people.\""
```

Comment:
This edit is good.
However it strikes me as a place for improvement, the descriptions of physical harm, etc., should probably be represented better. It seems weird to have them only on Material and Irreversible and not on Minimal. Can they be collected on just the description of the decision point, and not the options?

Reply:
Looks good. I will update the code to show Complex Decision points with required additional detailed information and hide it in the case of Options for the mouseover events.